tests/basic_one_way_send.go

```diff
@@ -71,7 +75,7 @@ func BasicOneWaySend() {
 		To:   &teleporterContractAddress,
 		Data: data,
 	}
-	result, err := subnetBInfo.ChainWSClient.CallContract(context.Background(), callMessage, nil)
+	result, err := subnetBInfo.ChainRPCClient.CallContract(context.Background(), callMessage, nil)
```

Comment:
What is the difference of using `ChainRPCClient` and `ChainWSClient` in here?

Reply:
The local network nodes are configured to accept both ws and rpc traffic on the same port, so the ws client can be used execlusively. On Fuji, however, nodes are often configured to separate ws and rpc connections on different ports, so we need two distinct clients in order for that to work. In general, the ws client should only be used for `eth_subscribe` or other sub methods, while the rpc clients should be used for any RPC API calls.